apps/fulfillment-e2e/src/integration/filters.cy.ts

```diff
@@ -0,0 +1,83 @@
+import { FiltersFragment } from '../support/page_fragments/filters.fragment';
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+const filtersFragment = new FiltersFragment();
+
+describe('When user opens the filters', () => {
+  const largeAndEarliestOrderFirst = (exists = true) =>
+    pickingListsFragment
+      .getPickingListsItems()
+      .eq(0)
+      .find('.total oryx-icon-button button')
+      .should(exists ? 'exist' : 'not.exist');
+
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.login();
+
+    filtersFragment.getFilterButtonTrigger().click();
+  });
+
+  it('should apply correct sorting configuration', () => {
+    //should open the filters modal
+    filtersFragment.getFiltersModal().should('have.attr', 'open');
```

Comment:
`.should('have.attr', 'open');` -> `.should('be.visible);`

A visibility check is better here because it is "user" centred and checks if block is ACTUALLY visible to user

`open` attribute check is not a prove of visibility. Element might have it, but still be not visible to the end user